app/Jobs/CleanupOrphanedPreviewContainersJob.php

```
@@ -0,0 +1,202 @@
+    }
+
+    public function handle(): void
+    {
+        try {
+            $servers = $this->getServersToCheck();
+
+            foreach ($servers as $server) {
+                $this->cleanupOrphanedContainersOnServer($server);
+            }
+        } catch (\Throwable $e) {
+            Log::error('CleanupOrphanedPreviewContainersJob failed: '.$e->getMessage());
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (undefined)

(StaticAccess)